config/development_polygon.toml

```diff
@@ -0,0 +1,216 @@
+[app]
+chain_id = 137
+
+# a database is optional. it is used for user authentication and accounting
+# TODO: how do we find the optimal db_max_connections? too high actually ends up being slower
+db_max_connections = 20
+# development runs cargo commands on the host and so uses "mysql://root:dev_web3_proxy@127.0.0.1:13306/dev_web3_proxy" for db_url
+# production runs inside docker and so uses "mysql://root:web3_proxy@db:3306/web3_proxy" for db_url
+db_url = "mysql://root:dev_web3_proxy@127.0.0.1:13306/dev_web3_proxy"
+
+# deposit_contract = "0xc9fcfa7e28ff320c49967f4522ebc709aa1fde7c"
+deposit_contract = "0x4e3bc2054788de923a04936c6addb99a05b0ea36"
+
+
+# a timeseries database is optional. it is used for making pretty graphs
+influxdb_host = "http://127.0.0.1:18086"
+influxdb_org = "dev_org"
+influxdb_token = "dev_web3_proxy_auth_token"
+influxdb_bucket = "dev_web3_proxy"
+
+# thundering herd protection
+# only mark a block as the head block if the sum of their soft limits is greater than or equal to min_sum_soft_limit
+min_sum_soft_limit = 1_000
+# only mark a block as the head block if the number of servers with it is great than or equal to min_synced_rpcs
+min_synced_rpcs = 1
+
+# redis is optional. it is used for rate limits set by `hard_limit`
+# TODO: how do we find the optimal redis_max_connections? too high actually ends up being slower
+volatile_redis_max_connections = 20
+# development runs cargo commands on the host and so uses "redis://127.0.0.1:16379/" for volatile_redis_url
+# production runs inside docker and so uses "redis://redis:6379/" for volatile_redis_url
+volatile_redis_url = "redis://127.0.0.1:16379/"
+
+# redirect_public_url is optional
+redirect_public_url = "https://llamanodes.com/public-rpc"
+# redirect_rpc_key_url is optional
+# it only does something if db_url is set
+redirect_rpc_key_url = "https://llamanodes.com/dashboard/keys?key={{rpc_key_id}}"
```

Comment:
We need to work with frontend team on this url again. I think they have a different format. Can you make a GitHub issue for this?